securedrop/journalist_templates/_sources_confirmation_final_modal.html

```diff
@@ -2,19 +2,19 @@
   <a href="#close" class="external" aria-label="{{ gettext('Close') }}"></a>
   <dialog open id="delete-confirm-menu-dialog" aria-labelledby="confirm-heading">
     <h2 id="confirm-heading" class="paragraph-spacing">
-      {{ gettext('When the account for a source is deleted:') }}
+      {{ gettext('When the account for a <abr title="user">source</abr>is deleted:') }}
```

Comment:
AFAICT the tag should be `<abbr title="...">` not `<abr title="...">` - unsure if both are valid.

Reply:
Looked it up and both are valid - https://www.w3schools.com/tags/tag_abbr.asp